tests/integration-tests-topologies/src/main/java/org/apache/pulsar/tests/topologies/PulsarCluster.java

```diff
@@ -112,6 +113,38 @@ private PulsarCluster(PulsarClusterSpec spec) {
             .withEnv("zookeeperServers", ZKContainer.NAME)
             .withEnv("configurationStoreServers", CSContainer.NAME + ":" + CS_PORT)
             .withEnv("clusterName", clusterName);
+
+        if(spec.enableTieredStorage()) {
+            this.s3Container = new S3Container(clusterName, S3Container.NAME)
+                .withNetwork(network)
+                .withNetworkAliases(S3Container.NAME)
+                .withEnv("initialBuckets", "pulsar-integtest");
```

Comment:
isn't this environment already configured in S3Container?

Reply:
your right will remove it